### ISSUE TITLE
error gulp serve

### ISSUE THREAD

**Issue by Yourdigitalheart** [yeoman/generator-webapp#652]
Iam using OSX Sierra 
nvm version
v6.10.1
Thanks for helping me

```
/node_modules/ws/lib/Receiver.js:306
  if (mask != null && buf != null) bufferUtil.unmask(buf, mask);
                                                                     ^

TypeError: Cannot read property 'unmask' of undefined
    at Receiver.unmask (/Users/insidebox/Documents/Dev-Insidebox/insidebox-front/node_modules/ws/lib/Receiver.js:306:46)
    at Receiver.finish (/Users/insidebox/Documents/Dev-Insidebox/insidebox-front/node_modules/ws/lib/Receiver.js:505:25)
    at Receiver.expectHandler (/Users/insidebox/Documents/Dev-Insidebox/insidebox-front/node_modules/ws/lib/Receiver.js:493:33)
    at Receiver.add (/Users/insidebox/Documents/Dev-Insidebox/insidebox-front/node_modules/ws/lib/Receiver.js:103:24)
    at Socket.realHandler (/Users/insidebox/Documents/Dev-Insidebox/insidebox-front/node_modules/ws/lib/WebSocket.js:825:20)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at TCP.onread (net.js:551:20)
```

**Comment by Yourdigitalheart** [comment]
Its come from yeoman itself

```
ParseError: Unexpected token '<' at 2:1 in https://registry.npmjs.org/-/_view/byKeyword?startkey=[%22yeoman-generator%22]&endkey=[%22yeoman-generator%22,%7B%7D]&group_level=3
<?xml version="1.0" encoding="utf-8"?>
^
    at /Users/insidebox/.nvm/versions/node/v6.10.1/lib/node_modules/yo/node_modules/got/index.js:120:14
    at BufferStream.<anonymous> (/Users/insidebox/.nvm/versions/node/v6.10.1/lib/node_modules/yo/node_modules/read-all-stream/index.js:64:3)
    at emitNone (events.js:91:20)
    at BufferStream.emit (events.js:185:7)
    at finishMaybe (/Users/insidebox/.nvm/versions/node/v6.10.1/lib/node_modules/yo/node_modules/readable-stream/lib/_stream_writable.js:511:14)
    at afterWrite (/Users/insidebox/.nvm/versions/node/v6.10.1/lib/node_modules/yo/node_modules/readable-stream/lib/_stream_writable.js:393:3)
    at _combinedTickCallback (internal/process/next_tick.js:86:20)
    at process._tickCallback (internal/process/next_tick.js:104:9)
```

**Comment by silvenon** [comment]
I have no idea what that error means. 😕 You can't run **any** Yeoman generator or just this one? Does this happen on a fresh install?

**Comment by neilhem** [comment]
Glad I would help, but I have never used this node module. Better ask support question in stackoverflow, you will get faster answer

**Comment by UlisesGascon** [comment]
I assume that this bug was fixed in #737. If not we can re-open it again 😃
